Meeting minutes 2021-09-02 (#1009)
HSF/hsf.github.io: organization/_posts/2021/2021-09-02-coordination.md

---
title: "HSF Coordination Meeting #213, 2 September 2021"
layout: plain_toc
---

> Present: Graeme Stewart, Josh McFayden, Attila Krasznahorkay, Kyle Knoepfel, Michel Jouvin, Pere Mato, Ben Morgan, Dorothea vom Bruch, Krzysztof Genser, Mark Neubauer, Caterina Doglioni, Eduardo Rodrigues, Allison Hall, Philippe Gras, Liz Sexton-Kennedy, Alexander Moreno, Andrei Gheata, Gordon Watts, Jerry Ling, Michael Hernandez, Nicola Skidmore, Sudhir Malik, Torre Wenaus, Daniel Elvira, Benedikt Hegner, Aman Goel

Reviewer: ```suggestion
Present: Graeme Stewart, Josh McFayden, Attila Krasznahorkay, Kyle Knoepfel, Michel Jouvin, Pere Mato, Ben Morgan, Dorothea vom Bruch, Krzysztof Genser, Mark Neubauer, Caterina Doglioni, Eduardo Rodrigues, Allison Hall, Philippe Gras, Liz Sexton-Kennedy, Alexander Moreno, Andrei Gheata, Gordon Watts, Jerry Ling, Michael Hernandez, Nicole Skidmore, Sudhir Malik, Torre Wenaus, Daniel Elvira, Benedikt Hegner, Aman Goel
```

Reviewer: "Nicole" is how the gets everyone to call her, hence this update (her ID name is Nicola indeed).

## News, general matters, announcements

### Julia Discussion

See slides on [the agenda](https://indico.cern.ch/event/1043626).

> - Workshop planned on **September 20** to discuss with more details how to move forward in the

Reviewer: ```suggestion
- Workshop planned on **September 20th** to discuss in more detail how to move forward in the
```

>   Julia evaluation planned in PyHEP WG

Reviewer: ```suggestion
  Julia evaluation planned within the PyHEP WG following a session on Julia at the PyHEP 2021 workshop
```

- [Generators](https://drive.google.com/file/d/18_ID_XLw4K1AjaYXuQC2pdNjZEWWbhk4/view?usp=sharing) (PDF copy)
- [Simulation](https://docs.google.com/document/d/1Rl9PH43gEWSzYtELT12-77zi-uW1avz8uClL44wcDvE/edit?usp=sharing) (Google Doc)
- [Foundation](https://docs.google.com/document/d/160o1LtpjL0Zgb4Suey1vyD_l03Ggh4frrADXMlOUcC4/edit?usp=sharing) (Google Doc)
- Analysis
  - [Data Science Tools](https://docs.google.com/document/d/13b1icyAsr99gPqgTQo5-Lso5x6aotdvVXe8lXjD5YvY/edit?usp=sharing) (Google Doc)
  - [ROOT](https://docs.google.com/document/d/15dlmSbOUCXXC35tOX5axNlK96MqnOqw_93RnSgCpDjI/edit?usp=sharing)
- [DOMA](https://www.overleaf.com/read/gkbppxdvcvvf)

Time for comments has basically passed, so if you want to give further feedback please do that ASAP.

#### LHCC Referees Meeting and Closed Session

LHCC Referees meeting was this week. Slides are [attached to the agenda](https://indico.cern.ch/event/1043626/). *Many thanks to everyone who gave input.*

Feedback:

- CernVM effort levels were discussed, stressing the need for projects like this to do *support* and *R&D*
- Some clarification on the ROOT/HighLO project, praised for attracting external funding

> - Discussion of the sustainability of the current C++ course, where - stressed that others (beyond Sebastien P) were contributing and the course material was open source

Reviewer 2: that sentence is not understandable

- Long discussion on the effort available for software and computing, particularly for computing operations, the fact that this is declining and not well recognised
  - Institute commitment was seen to be key to solving this
  - Recognised as part of a wider problem, covering almost all areas

>   - Software is not in quite such a parlous state as operations, but it was stresses that software needs *long term support* and *viable career paths* - just funding cool R&D isn't enough

Reviewer: ```suggestion
  - Software is not in quite such a parlous state as operations, but it was stressed that software needs *long term support* and *viable career paths* - just funding cool R&D isn't enough
```

### Software Training

- Organised the first [virtual Software Carpentry workshop](https://indico.cern.ch/event/1058873/) 25-27 Aug 2021 (75 participants registered)
  - Software Carpentry workshops (SCW) will be organized every ~3 months.

- First publication on Software training (based on vCHEP 2021 contribution) submitted (1 Aug 2021) to CSBS  - Springer (Computing and Software for Big Science) [arXiv version](https://arxiv.org/abs/2103.00659))

#### 3rd HEP C++ Course and Hands-on Training

- [Event](https://indico.cern.ch/event/1019089/) is running this week. 75 students attending. Another 90 on the waiting list. With this course we will have trained 200 students since October 2020.
- Many thanks to all lecturers and mentors!!!
- Meeting notes that we should improve the connection between the C++ teaching and the WG.

### Event Generators

- Not much to report. Very little action in August.
  - Main focus now is on implementing comments received on the LHCC document and recirculating to the WG.
  - "Taming the accuracy of event generators" workshop last week, will be interesting for us to look at discussions: <https://indico.cern.ch/event/999271/>

### Frameworks

> - Will (hopefully) have a meeting next week after a longer hiatus. (<https://indico.cern.ch/event/1072798/>) With the intention of going back to regular meetings in the autumn once again.

Reviewer: ```suggestion
- Will (hopefully) have a meeting next week after a longer hiatus (<https://indico.cern.ch/event/1072798/>). With the intention of going back to regular meetings in the autumn once again.
```

---

## AOB

### HSF Domain Name

Ticket still open, no recent movement.

### SIDIS COST Proposal

We are currently working on a funding proposal (EU COST) in the context of the "Software Institute for Data Intensive Sciences", aiming to bridge the gap between computer scientists and natural science software engineers. The "grand theme" of the proposal will be around the topic of "software sustainability".

> For a success proposal it is useful to include participants also from "inclusive target" countries (mostly eastern European countries). In case you have contacts in this direction please get in contact with Graeme or Stefan.

Reviewer: ```suggestion
For a successul proposal it is useful to include participants also from "inclusive target" countries (mostly eastern European countries). In case you have contacts in this direction please get in contact with Graeme or Stefan.
```

Reviewer 2: *successful*
